docs/maintainers/ci.md

```diff
@@ -1,8 +1,17 @@
-# Concept CI
+# Continuous Integration
+
+## Table of Contents
```

Comment:
I imagine this file could grow quite big, so I added a ToC. I'm happy to remove it if it's not wanted, though.

Reply:
Yeah that could be. Maybe we should split this into multiple files later. We do already have some pretty big files, so I don't feel it is very bad.